src/renderer/widgets/AddProxyModal/lib/submit-proxy-utils.ts

```diff
@@ -0,0 +1,19 @@
+import { SubmitStep } from './types';
+
+export const submitProxyUtils = {
+  isLoadingStep,
+  isSuccessStep,
+  isErrorStep,
+};
+
+function isLoadingStep(step: SubmitStep): boolean {
```

Comment:
seems it's not only proxy submitting steps.

Reply:
this file contains utility functions only for `Submit` step